Java.gitignore

```diff
@@ -10,6 +18,12 @@
 *.jar
 *.war
 *.ear
+*.zip
+*.tar.gz
+*.rar
+
+# Vagrant
```

Comment:
We have an existing [`Global\Vagrant.gitignore`](https://github.com/github/gitignore/blob/master/Global/Vagrant.gitignore). Please drop these rules.